testscripts/fuzz_help.txt

```diff
@@ -0,0 +1,8 @@
+# Start by seeing if anything works at all.
+# This is intended to excercise travis and the testscript cmd invoking go-fuzz.
+# If this fails, likely everything else will fail too, but if so might be easier to 
+# start by troubleshooting this.
+
+# Ask go-fuzz-build to emit its help message, which should be a failing status code.
+! exec go-fuzz-build -h
```

Comment:
the other big advantage of TestScript is that you can replace `exec go-fuzz-whatever` with just `go-fuzz-whatever`, meaning that the tests don't depend on $PATH, and you can more easily get instrumentation like code coverage.

Reply:
See earlier comment about using the testscript binary. `exec` here is a conscious choice for consistency. Other approaches are of course possible.

Reply:
SGTM